www/js/my-app.js

```diff
@@ -114,32 +114,34 @@ function init() {
 function searchSubmit(e) {
   var formData = myApp.formToJSON('#search');
   e.preventDefault();
-  if (!formData.q) {
+  if (!formData.term) {
     myApp.alert('Please enter a search term', 'Search Error');
     return;
   }
 
   if (formData.filter === 'all') {
-    formData.q = formData.q.trim();
+    formData.term = formData.term.trim();
   } else {
-    formData.q = formData.filter + ':' + formData.q.trim();
+    formData.term = formData.filter + ':' + formData.term.trim();
   }
   delete formData.filter;
-  formData.type = 'track';
+  formData.media = 'music';
   $$('input').blur();
   myApp.showPreloader('Searching');
   $$.ajax({
     dataType: 'json',
     data: formData,
     processData: true,
-    url: 'https://api.spotify.com/v1/search',
+    url: 'https://itunes.apple.com/search',
```

Comment:
You'll probably want to update the CSP in `index.html` as well for when not running in the developer app, etc.